README.md

```diff
@@ -208,6 +208,7 @@ const columns = [
 |:--:|:-----|:--|:-----|
 |**`display`**|string|'true'|Display column in table. `enum('true', 'false', 'excluded')`
 |**`empty`**|boolean|false|This denotes whether the column has data or not (for use with intentionally empty columns)
+|**`viewColumns`**|boolean|true|Allow user to toggle column visibility through 'View Column' list
```

Comment:
I like the added explanation, but to keep this language parallel with the other similar options, how about `Show/hide viewColumns icon from toolbar, which allows the user to...`

Reply:
@gabrielliwerant - this option doesn't affect the icon display. It can be confusing because the identically named option exists for the whole table (as well as individual columns). The former sets the icon, the latter is for the list that opens when the icon is clicked.

To add to the confusion, this option sets whether the column name should be shown/hidden from the list that shows/hides the column itself. 
Having said that, how about:
"Allow user to show/hide the column name in the 'View Column' list."

Reply:
Oh my, you are right, thanks for clarifying that. Given that, I have no issues with your description.

Seems like this option could use a better name. Something like `hideable`? Would you want to make those changes here? If not, that's fine, and we can move this one forward as is.

Reply:
It would be considered a breaking change if a site is using the existing option.
As such, it would need to also be documented as part of future release notes.

Personally, I'm ok to continue using the option as provided, and was just happy to discover it is already possible to do this.

Reply:
Agreed, it should be noted and bumped in version accordingly. I was planning on leaving that note in the PR if you wanted to move forward with it, but it's not critical.

Reply:
Sure, I can make the change to the column level option name, we should just agree on what it should be.
`hideable` is not appropriate because it will determine both if a column can be hidden (if visible) or visible (if hidden).

How about `allowToggle`?